group-membership-form/src/Code.js

```diff
@@ -0,0 +1,88 @@
+var ADDED_TO_GROUP_SUBJECT = 'Added to group';
+var ADDED_TO_GROUP_DOC_URL = 'https://docs.google.com/document/d/1-ajkkIP8gUWqMcnpXhkqwlM_2Y18USLdJ-pFZdDEZ70/edit?usp=sharing';
+
+var ALREADY_IN_GROUP_SUBJECT = 'Already in group';
+var ALREADY_IN_GROUP_DOC_URL = 'https://docs.google.com/document/d/11AO7vwk6179ohuxGO_NXSoDB0m_H5e-5XEtwiWRVNOM/edit?usp=sharing';
```

Comment:
Does this need a separate doc?

Reply:
Yes, for simplicity I have them separated. Based on the logic if a user is newly added they get a welcome letter and if they were already added they get an email that says they were already added. Didn't want it to get go silent.